src/config.ts

```diff
@@ -110,6 +115,12 @@ export function parseConfig(): DeploymentsConfig | DeploymentStackConfig {
   const description = getOptionalStringInput("description");
   const tags = getOptionalStringDictionaryInput("tags");
   const maskedOutputs = getOptionalStringArrayInput("masked-outputs");
+  const cloud =
+    getOptionalEnumInput("cloud", [
+      "azureChinaCloud",
+      "azureUSGovernment",
+      "azureGermanCloud",
```

Comment:
```suggestion
      "azureCloud",
      "azureChinaCloud",
      "azureUSGovernment",
      "azureGermanCloud",
```

[nit] Doesn't hurt to allow the user to pass in "azureCloud" explicitly if they choose to.